sql/analyzer/triggers.go

```diff
@@ -440,14 +440,14 @@ func getTriggerLogic(ctx *sql.Context, a *Analyzer, n sql.Node, scope *plan.Scop
 				),
 			)
 		}
-		s := (*plan.Scope)(nil).NewScope(scopeNode).WithMemos(scope.Memo(n).MemoNodes()).WithProcedureCache(scope.ProcedureCache())
+		s := scope.NewScope(scopeNode)
```

Comment:
some sort of comment explaining this would be helpful. i never understood what memos/proc cache did, i wonder if we can delete those off the `*Scope` object

Reply:
I deleted `WithMemos()` and `WithProcedureCache()` because `NewScope()` already copies them from the parent, but it doesn't look like either of those are used for anything useful.

I'll look more closely and see if they are really required.

Reply:
It turns out `ProcedureCache` contains user defined stored procedures and `Memos` are used to determien if there are circular dependencies in triggers, so neither one can be deleted.